docker-compose.yml

```diff
@@ -24,7 +24,7 @@ services:
       MINIO_SECRET_KEY: minio1234
     command: server --address ":9001" /data
   db:
-    image: postgres:9.6
+    image: mdillon/postgis:9.6
```

Comment:
Steve and Derek also commented on this - could we have this be a build: - and not rely on personal repo?